web/templates/email/owner_add.html.eex

```diff
@@ -0,0 +1,6 @@
+<p><%= @email %> has been added to <%= @package %>.</p>
```

Comment:
Maybe this would be better expressed as:
"*username* (*email*) has been added as an owner to the package *package*"?